kubernetes/app-with-provider/__test__/answers-aks.yaml

```diff
@@ -8,6 +8,7 @@ ClientID: A
 ClientSecret: B
 SubscriptionID: C
 TenantID: D
+GenerateDockerComposeSetup: true
```

Comment:
Strangely getting this error:
`$ xl blueprint -l ../../../LOVE-1096 -b kubernetes/app-with-provider -a ../../../LOVE-1096/kubernetes/app-with-provider/__test__/answers-aks.yaml`
`[dataPrep] Using answer file value [AKS] for variable [Provider]`
`[dataPrep] Using answer file value [true] for variable [KubeApp]`
`Error while creating Blueprint: no variable found in list by name [GenerateDockerComposeSetup]`

I'm using the `xl` binary with the blueprints repo

Reply:
The binary included in this PR should work as it fixes some bug with composability